### PR TITLE
Improve the information about updating off-line

### PR DESCRIPTION
**Pull request by yecril71pl**
# PR Summary

The previous information was self-contradictory.

## PR Context

Select the area of the Table of Contents containing the documents being changed.

**Conceptual content**
- [ ] Overview and Install
- [ ] Learning PowerShell
  - [ ] PowerShell 101
  - [ ] Deep dives
  - [ ] Remoting
- [ ] Release notes (What's New)
- [ ] Windows PowerShell
  - WMF, ISE, release notes, etc.
- [ ] DSC articles
- [ ] Community resources
- [ ] Sample scripts
- [ ] Gallery articles
- [ ] Scripting and development
  - [ ] Legacy SDK

**Cmdlet reference & about_ topics**
- [X] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

## PR Checklist

- [X] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [X] PR has a meaningful title
- [X] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [X] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/7.1/Microsoft.PowerShell.Core/Update-Help.md [hunk 1]
```diff
@@ -50,23 +50,24 @@ current session are included. PowerShell modules are stored in a location listed
...
+You can also use `Update-Help` on computers that are not connected to the Internet. 
+In order to do so, use the `Save-Help`cmdlet to download help files from the Internet 
+and save them in a file system location, such as a shared folder or file system directory. 
+Then use the `SourcePath` parameter of `Update-Help` to download the updated help files 
+from a file system location and install them on the computer.
 
 You can automate help updates by adding the `Update-Help` cmdlet to your PowerShell profile. By
 default, `Update-Help` runs only one time per day on each computer. To override the once-per-day
-limit, use the **Force** parameter.
+limit, use the `Force` parameter.
```
Comment (sdwheeler) [inline reference/7.1/Microsoft.PowerShell.Core/Update-Help.md:64]: Parameter names are bolded. See https://docs.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide?view=powershell-7#formatting-command-syntax-elements

```suggestion
limit, use the **Force** parameter.
```

**Comment by yecril71pl** [comment]
What other versions?

**Comment by sdwheeler** [comment]
Apply the same format and wording changes to v5.1,v6, and v7.0 versions of the article.

**Comment by yecril71pl** [comment]
The changes do not match 7.0.

**Pull request merged**